src/index.ts

```diff
@@ -1,120 +1,19 @@
 import { Octokit } from "@octokit/core";
+import {
+  CreateOrUpdateUniqueIssueOptionsT,
+  CreateOrUpdateUniqueIssueResponseT,
+} from "./types";
 import { VERSION } from "./version";
-import { Endpoints } from "@octokit/types";
+import { composeUniqueIssue } from "./compose-unique-issue";
 
-const OCTOKIT_UNIQUE_ISSUE_ID_PREFIX = "octokit-unique-issue id=";
-
-type CreateOrUpdateUniqueIssueOptionsT =
-  Endpoints["POST /repos/{owner}/{repo}/issues"]["parameters"] & {
-    identifier: string;
-    close_previous?: boolean;
-  };
-
-type CreatedIssueResponseT =
-  Endpoints["POST /repos/{owner}/{repo}/issues"]["response"] & {
-    updated: false;
-    closed_issues: Endpoints["GET /search/issues"]["response"]["data"]["items"];
-  };
-type UpdatedIssueResponseT =
-  Endpoints["PATCH /repos/{owner}/{repo}/issues/{issue_number}"]["response"] & {
-    updated: true;
-    closed_issues: [...[]];
-  };
-type CreateOrUpdateUniqueIssueResponseT =
-  | CreatedIssueResponseT
-  | UpdatedIssueResponseT;
+export { composeUniqueIssue } from "./compose-unique-issue";
```

Comment:
the coverage calculation isn't happy after this change because this export is not covered. before i put in the effort to resolve the coverage failure, i wanted to make sure i'd be headed in the direction you prefer.

currently, the logic of this export is tested through the `uniqueIssue` interface, but not directly. it would be natural enough to move the existing tests to test the export from the `compose-unique-issue.ts` file directly, but then i'm curious how you'd prefer to testing the behaviors of the `index.ts` file. do you have a convention you normally follow for that situation?

Reply:
I would add a test to the smoke tests and a new `describe("composeCreateOrUpdateUniqueIssue", () => {` block to the existing test file